src/main/resources/com/google/api/codegen/go/mock.snip

```diff
@@ -0,0 +1,129 @@
+@extends "go/header.snip"
+
+@snippet generate(view)
+    {@headerComment(view.fileHeader)}
+
+    package {@view.fileHeader.localPackageName}
+
+    import (
+        {@imports(view.fileHeader.imports)}
```

Comment:
Why don't you have a snippet in header.snip which does all of the header comment, package, and imports?